dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -619,8 +620,11 @@ private void useRegistryForConfigIfNecessary() {
             Environment.getInstance().getDynamicConfiguration().orElseGet(() -> {
                 ConfigManager configManager = ConfigManager.getInstance();
                 ConfigCenterConfig cc = configManager.getConfigCenter().orElse(new ConfigCenterConfig());
-                cc.setParameters(new HashMap<>());
-                cc.getParameters().put(org.apache.dubbo.remoting.Constants.CLIENT_KEY,rc.getClient());
+                if (StringUtils.isNotEmpty(rc.getClient())) {
+                    Map<String, String> newParametersMap = cc.getParameters() != null ? cc.getParameters() : new HashMap<>();
+                    newParametersMap.put(CLIENT_KEY, rc.getClient());
+                    cc.setParameters(newParametersMap);
+                }
```

Comment:
ok. i wil commit again soon

Reply:
I am sorry that I made a mistake about the issue. If we need check if the ConfigCenterConfig instance have the parameters of non-null value, we also need to check if it has protocol and address with non-empty value. But it is redundant to check these parameters in this situation because the ConfigCenterConfig is a new instance.
So I think we don't need to do so. And this pr is not needed.
I am really sorry about this issue.